packages/fluentui/react-builder/src/config.tsx

```diff
@@ -538,6 +548,85 @@ export const renderJSONTreeToJSXElement = (
   });
 };
 
+const packageImportList: Record<string, CodeSandboxImport> = {
+  '@fluentui/react-icons-northstar': {
+    version: projectPackageJson.version,
+    module: FUIIcons,
```

Comment:
```suggestion
```

We don't need `module` there. If it's required, let's make it optional in the type